bukuserver/README.md

```diff
@@ -1,3 +1,32 @@
+#### Install server
+You need to have some packages before you install `bukuserver` on your server. So be sure to have `python3`, `python3-pip` , `python3-dev`, `libffi-dev` packages from your distribution.
+##### Installing PIP, virtualenv and dependencies
+```
+$ python3 -m pip install --user --upgrade pip
+$ python3 -m pip install --user virtualenv
+$ python3 -m virtualenv env
+$ source env/bin/activate
+(env) $ pip install appdirs
```

Comment:
What is (env)?

Reply:
it mean virtual environment is activated.

imo it should be removed and let the user decide their own virtual env

Reply:
Yes